pkg/operator/hive/hive.go

```diff
@@ -61,6 +61,11 @@ func (r *ReconcileHiveConfig) deployHive(hLog log.FieldLogger, h *resource.Helpe
 		hiveDeployment.Spec.Template.Spec.Containers[0].Env = append(hiveDeployment.Spec.Template.Spec.Containers[0].Env, dnsServersEnvVar)
 	}
 
+	if instance.Spec.Backup.Velero.Enabled {
+		hLog.Infof("Velero Backup Enabled.")
+		hiveDeployment.Spec.Template.Spec.Containers[0].Command = append(hiveDeployment.Spec.Template.Spec.Containers[0].Command, "--velero-backup")
+	}
```

Comment:
This is slightly different from how we're passing most things from hiveconfig to hive via env vars. Any thoughts on which approach is better and when/where we should draw the line between the two? I kinda like the CLI as opposed to env vars as I can run the binary with --help to see options, rather than having env vars embedded all over the code.

Right now the env vars are: global pull secret, zone check dns servers.

Curious what the team thinks. cc @staebler @abutcher @LalatenduMohanty @joelddiaz

Reply:
I'm happy to do it either way. I wasn't trying to "blaze a new trail" here, I just didn't see the other way.

I'll do it the env var way for now and then if we decide to do CLI options, I'll add this back.

Reply:
Ok FWIW I kinda like it. :)

Reply:
I like the env var way as does not tie us with the CLI strongly. However I am not very sure which will be maintainable long term.